tests/src/test/scala/cats/tests/OptionTSuite.scala

```diff
@@ -70,6 +71,23 @@ class OptionTSuite extends CatsSuite {
     checkAll("Functor[OptionT[ListWrapper, ?]]", SerializableTests.serializable(Functor[OptionT[ListWrapper, ?]]))
   }
 
+  {
+    // F has an Invariant
+    Invariant[Show]
+    Invariant[OptionT[Show, ?]]
+
+    checkAll("OptionT[Show, ?]", InvariantTests[OptionT[Show, ?]].invariant[Int, Int, Int])
+    checkAll("Invariant[OptionT[Show, ?]]", SerializableTests.serializable(Invariant[OptionT[Show, ?]]))
+  }
```

Comment:
Ideally we'd have a check here for a type that has an `Invariant` instance but not a `Contravariant` instance to make sure that we cover this use-case. I think that `Semigroup` would be an example of such a type.